content-src/asrouter/templates/FirstRun/FirstRun.jsx

```diff
@@ -7,6 +7,9 @@ import { Interrupt } from "./Interrupt";
 import { Triplets } from "./Triplets";
 import { BASE_PARAMS } from "./addUtmParams";
 
+// Note: should match the transition time on .trailheadCards in _Trailhead.scss
+const TRANSITION_LENGTH = 500;
```

Comment:
Probably good to add a matching comment in the scss:
https://github.com/mozilla/activity-stream/blob/469c7ee75d8fb4ddd64784ef058d79284584acf7/content-src/asrouter/templates/Trailhead/_Trailhead.scss#L268